### DISCUSSION TITLE
alas老是出错是怎么回事呢？

### DISCUSSION THREAD

**Discussion by tk41104** [LmeSzinc/AzurLaneAutoScript#1507] [Q&A] [answered]
昨天都好好的，今天我alas设置什么都没改，游戏里我也什么都没动，中间也没提停止运行，今天早上突然就报错了

CRITICAL 14:09:45.573 │ Failed to enter d2, too many click on d2                   
CRITICAL 14:09:45.575 │ Possible reason #1: You haven't reached the commander level
         to unlock this stage.                                                     
CRITICAL 14:09:45.577 │ Request human takeover
指挥官等级已经130级了，运行到点开d2，然后就停了，也不点出击按钮，再次点击运行还是错在这里，，，不知道咋回事，，，
有大佬解答一下吗，谢谢

**Comment by tk41104** [comment]
D2昨天alas都刷的好好的是正常的

**Comment by LmeSzinc** [comment, marked as answer]
#1510
